core/src/main/java/com/linecorp/armeria/server/annotation/processor/DocumentationProcessor.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.annotation.processor;
```

Comment:
I think we should move this to a separate artifact - some build systems will automatically run all annotation processors on classpath but it's probably better for this to be opt-in like spring boot configuration process

Reply:
+1 - Let me wait for a follow-up commit.

Reply:
I've moved it into its own project. I hope I didn't make any mistakes 🙏